linter/find_variables.go

```diff
@@ -5,10 +5,10 @@ import (
 	"github.com/google/go-jsonnet/parser"
 )
 
-type vScope map[ast.Identifier]*Variable
+type vScope map[ast.Identifier]*variable
```

Comment:
Variable wasn't used outside the package so I changed it to a private symbol. If the intent is to make it public somewhere down the line, I can add doc-comments instead.

Reply:
Let's keep it private for now, that's ok.

(I think I had some plans for it being used in the future documentation generator or something, but I'm not sure, no point in keeping stuff public "just in case".)